libs/sdk-ui-dashboard/src/converters/filterConverters.ts

```diff
@@ -23,28 +23,30 @@ import { IDashboardFilter } from "../types";
 /**
  * Gets {@link IDashboardFilter} items for filters specified in given filterContext in relation to the given widget.
  *
+ * forWidget...
```

Comment:
Maybe missing something here?

Reply:
This doc line has been removed. Forgotten "helping line"